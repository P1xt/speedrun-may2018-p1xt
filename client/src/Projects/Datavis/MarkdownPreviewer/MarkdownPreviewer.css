.MarkdownPreviewer {
  text-align: left;
  margin-top: 40px; }
  .MarkdownPreviewer > h1 {
    text-transform: none; }
