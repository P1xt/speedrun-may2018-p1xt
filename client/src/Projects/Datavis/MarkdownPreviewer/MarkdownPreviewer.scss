.MarkdownPreviewer {
  text-align: left;
  margin-top: 40px;

  &>h1 {
    text-transform: none
  }
}
