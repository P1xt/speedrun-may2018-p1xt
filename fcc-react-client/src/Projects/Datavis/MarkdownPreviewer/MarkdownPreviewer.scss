.MarkdownPreviewer {
  text-align: left;
  margin-top: 40px;

  & > h1 {
    text-transform: none;
  }
  &_input {
    textarea {
      min-height: 600px;
    }
  }
  &_output {
    padding: 20px;
    background: #ffffff;
    height: 600px;
    overflow: scroll;

    table {
      tr {
      }
      td {
        border: 1px solid #cccccc;
      }
    }
  }
}
