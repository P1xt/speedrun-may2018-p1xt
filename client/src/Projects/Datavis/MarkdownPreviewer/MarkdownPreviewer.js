import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./MarkdownPreviewer.css";

const MarkdownPreviewer = () => {
  return (
    <div className="MarkdownPreviewer">
      <Container>
        <Row>
          <Col>
            <h1>Markdown Previewer</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default MarkdownPreviewer;
