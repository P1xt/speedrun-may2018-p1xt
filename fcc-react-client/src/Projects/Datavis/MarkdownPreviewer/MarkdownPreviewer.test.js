import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import MarkdownPreviewer from "./MarkdownPreviewer";

describe("<MarkdownPreviewer />", () => {
  it("renders without crashing", () => {
    shallow(<MarkdownPreviewer />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<MarkdownPreviewer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
