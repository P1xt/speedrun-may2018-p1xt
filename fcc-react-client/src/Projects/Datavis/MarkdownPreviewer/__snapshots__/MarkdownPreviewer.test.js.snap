// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarkdownPreviewer /> renders correctly 1`] = `
<div
  className="MarkdownPreviewer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-12 col-sm-6"
      >
        <h2>
          Enter Markdown:
        </h2>
        <hr />
        <div
          className="MarkdownPreviewer_input"
        >
          <textarea
            autoFocus={true}
            className="textInput form-control"
            onChange={[Function]}
            value="# Put markdown here"
          />
        </div>
      </div>
      <div
        className="col-12 col-sm-6"
      >
        <h2>
          Formatted Result:
        </h2>
        <hr />
        <div
          className="MarkdownPreviewer_output"
        >
          <div
            dangerouslySetInnerHTML={
              Object {
                "__html": "<h1 id=\\"put-markdown-here\\">Put markdown here</h1>
              ",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
